spec/requests/data_api/tad_exports_spec.rb

```diff
@@ -5,18 +5,79 @@
 
   it_behaves_like 'a TAD API endpoint', '/'
 
+  it 'allows access to the API for TAD users' do
+    get_api_request "/data-api/tad-data-exports?updated_since=#{CGI.escape(1.month.ago.iso8601)}", token: tad_api_token
+
+    expect(response).to have_http_status(:success)
+    expect(parsed_response).to be_valid_against_openapi_schema('TADDataExportList')
+  end
+
   it 'returns a list of data exports' do
-    create(:submitted_application_choice, :with_completed_application_form, status: 'rejected')
+    get_api_request "/data-api/tad-data-exports?updated_since=#{CGI.escape(1.day.ago.iso8601)}", token: tad_api_token
+
+    expect(response).to have_http_status(:success)
+    expect(parsed_response).to be_valid_against_openapi_schema('TADDataExportList')
+  end
+
+  context 'when data export is exported' do
+    before do
+      create(:submitted_application_choice, :with_completed_application_form)
+
+      data_export = DataExport.create!(name: 'Daily export of applications for TAD', export_type: :tad_applications)
+      DataExporter.perform_async(DataAPI::TADExport, data_export.id)
+    end
+
+    it 'returns data exports response JSON values as expected' do
+      data_export = DataExport.last
+
+      get_api_request "/data-api/tad-data-exports?updated_since=#{CGI.escape(1.day.ago.iso8601)}", token: tad_api_token
 
-    data_export = DataExport.create!(name: 'Daily export of applications for TAD')
-    DataExporter.perform_async(DataAPI::TADExport, data_export.id)
+      response_data = parsed_response.dig('data', 0)
 
-    data_export = DataExport.create!(name: 'Daily export of applications for TAD')
-    DataExporter.perform_async(DataAPI::TADExport, data_export.id)
+      expect(response_data['export_date'].to_time.iso8601).to eq(Time.zone.now.iso8601)
```

Comment:
Fine — call to standard library like `Time` feels fine — it's when we call our own code that we ought to be careful! 🙂

Reply:
i guess i was wondering whether the `export_date/completed_at` time set in the before block would always match up with `Time.zone.now` or whether it could be a few milliseconds out

Reply:
Good q — could use `be_within`: https://blog.eq8.eu/article/rspec-be-within-matcher.html

Reply:
Thanks, fixed in 69dfbc7ebf9889d9d0d30a9cc17e5f332cb2a469